groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -222,21 +222,17 @@ extern const Optional_OptNoSuchType optNoSuchType;
 
 #ifdef BSLS_LIBRARYFEATURES_HAS_CPP11_BASELINE_LIBRARY
 
-template <class U, class V, bool D>
-struct Optional_IsConstructible : std::is_constructible<U, V> {
-    // This metafunction is derived from 'std::is_constructible<U, V>' in
-    // C++11 and later and from 'bsl::integral_constant<D>' in C++03 with the
-    // value of 'D' typically chosen to not affect the constraint this
-    // metafunction appears in.
-};
+// This macro is defined as 'std::is_constructible<U, V>' in C++11 and later,
+// and as 'bsl::integral_constant<DEFAULT>' in C++03 with the value of
+// 'DEFAULT' typically chosen to not affect the constraint this macro appears
+// in.
+#define BSLSTL_OPTIONAL_IS_CONSTRUCTIBLE(U, V, DEFAULT)                       \
+    std::is_constructible<U, V>
```

Comment:
```suggestion
    std::is_constructible<U, V>::value
```

Reply:
done, applied throughout,  also modified the macro names to end with _V